### DISCUSSION TITLE
Adding column to collection overview

### DISCUSSION THREAD

**Discussion by crackAT** [statamic/cms#9931] [Q&A]
Hei all, 
is there a way to add entry id as column to the collection overview where entries of a collection are listed? 

Thanks

**Comment by sjclark** [comment]
Any particular reason?

I tried adding it as a `listable` field in a blueprint but received `Field with a handle of id is a reserved word.`

**Comment by edalzell** [comment]
maybe a computed field?

**Reply by sjclark** [reply]
I was going to suggest that but for some reason I thought you couldn’t sort or list based off computed fields?
